src/Controls/src/Core/Shapes/BorderShapeTypeConverter.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.ComponentModel;
+using System.Globalization;
+using Microsoft.Maui.Graphics.Converters;
+using System.Linq;
+
+namespace Microsoft.Maui.Controls.Shapes
+{
+	public class BorderShapeTypeConverter : TypeConverter
+	{
+		public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
+			   => sourceType == typeof(string);
+
+		public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
+		{
+			var strValue = value?.ToString();
+			string[] corners = strValue.Split(new char[] { ' ', ',' });
+
+			if(corners.Length > 0)
+			{
+				var cornerRadius = strValue.Split(',');
+				if (cornerRadius.Length == 4
+					&& double.TryParse(cornerRadius[0], NumberStyles.Number, CultureInfo.InvariantCulture, out double tl)
+					&& double.TryParse(cornerRadius[1], NumberStyles.Number, CultureInfo.InvariantCulture, out double tr)
+					&& double.TryParse(cornerRadius[2], NumberStyles.Number, CultureInfo.InvariantCulture, out double bl)
+					&& double.TryParse(cornerRadius[3], NumberStyles.Number, CultureInfo.InvariantCulture, out double br))
+					return new RoundRectangle { CornerRadius = new CornerRadius(tl, tr, bl, br) };
+				if (cornerRadius.Length > 1
```

Comment:
Should this be >= 1? If we only use the first value specified, we want to only care if it's at least 1 number.
```suggestion
				if (cornerRadius.Length >= 1
```